packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -32,15 +32,22 @@ export default class QuanticCategoryFacet extends LightningElement {
   @api engineId;
   /** @type {boolean} */
   @api noSearch = false;
-
+  /** @type {boolean} */
+  @api get isCollapsed() {
```

Comment:
Why do you need the `@api` decorator here?

Reply:
is-collapsed can be specified in the html tag for 
```
<c-quantic-category-facet is-collapsed></c-quantic-category-facet>
```
in case a search interface should have a facet collapsed by default. Without the decorator it won't work.

Reply:
I was expecting the `@api` decorator on the setter for the reason you explained. I guess LWC applies the decorator to both the getter and the setter in this case. Good to know.

Reply:
Ah yes well I had tried out too and found out the hard way that it crashes the component 😅